compiler/rustc_mir/src/transform/promote_consts.rs

```diff
@@ -646,16 +648,18 @@ impl<'tcx> Validator<'_, 'tcx> {
                         });
                     }
                 }
-                Err(Unpromotable)
+                return Err(Unpromotable);
             }
 
             Rvalue::Ref(_, kind, place) => {
                 // Special-case reborrows to be more like a copy of the reference.
                 let mut place_simplified = place.as_ref();
                 if let [proj_base @ .., ProjectionElem::Deref] = &place_simplified.projection {
-                    let base_ty = Place::ty_from(place_simplified.local, proj_base, self.body, self.tcx).ty;
+                    let base_ty =
+                        Place::ty_from(place_simplified.local, proj_base, self.body, self.tcx).ty;
```

Comment:
huh what happened here?

Reply:
Just `rustfmt`, or is there any other difference that I am missing?

Reply:
yea, but I was confused why rustfmt would do this.

Reply:
Because it is being `rustfmt`? ;)

The variable name got longer, since I need to keep the original `place` around. As a result this line crossed the threshold.

Reply:
oh... this is due to an intermediate state that was reverted later? Because this specific line is *exactly* the same (modulo whitespace) as the line that was there before.

Reply:
Maybe I didn't fmt between all commits.